api/blocks/blocks_test.go

```diff
@@ -67,6 +71,22 @@ func testBadQueryParams(t *testing.T) {
 
 	assert.Equal(t, http.StatusBadRequest, statusCode)
 	assert.Equal(t, "expanded: should be boolean", strings.TrimSpace(string(res)))
+
+	badQueryParams = "?raw=1"
+	res, statusCode, err = tclient.RawHTTPClient().RawHTTPGet("/blocks/best" + badQueryParams)
+	require.NoError(t, err)
+
+	assert.Equal(t, http.StatusBadRequest, statusCode)
+	assert.Equal(t, "raw: should be boolean", strings.TrimSpace(string(res)))
+}
+
+func testMutuallyExclusiveQueries(t *testing.T) {
+	badQueryParams := "?expanded=true&raw=true"
```

Comment:
maybe we can add also combinations here:

- expanded = true, raw. = false
- expanded = false, raw = true
- expanded = false, raw = false

Reply:
@leszek-vechain I have checked the code, there are already checks for that, was that what you mean?
https://github.com/vechain/thor/blob/96634c89c54c4712384c6d00ecf955e03e2ca345/api/blocks/blocks_test.go#L84
https://github.com/vechain/thor/blob/96634c89c54c4712384c6d00ecf955e03e2ca345/api/blocks/blocks_test.go#L93
https://github.com/vechain/thor/blob/96634c89c54c4712384c6d00ecf955e03e2ca345/api/blocks/blocks_test.go#L104
https://github.com/vechain/thor/blob/96634c89c54c4712384c6d00ecf955e03e2ca345/api/blocks/blocks_test.go#L194

Reply:
yeah nice :)